package.json

```diff
@@ -4,8 +4,8 @@
     "postinstall": "node script/get-quasar.js",
     "clean": "rimraf output && rimraf .pulp-cache && rimraf test/tmp",
     "build": "pulp build -- --censor-lib --strict",
-    "property-tests": "pulp test --test-path test/src --main Test.Property.Main",
-    "test": "pulp test --test-path test/src --main Test.Property.Main && pulp test --test-path test/src"
+    "test-fast": "pulp test --test-path test/src --main Test.Fast",
```

Comment:
i like the idea that `--argument` is for scripts and `-a` is for manual typing. its not super important for now but maybe we could supply both shorthand and full hand for commonly used scripts?

```
"test-implementation": "pulp test --test-path test/src --main Test.Fast",
"ti": "npm run test-implementation"
```